README.md

```diff
@@ -14,16 +14,25 @@ $ pip install -r requirements.txt
 
 ## Usage:
 
+The following environment variables must be set:
+* `TIMEOUT`: The timeout (in seconds) for the request made to Contile
+* `TARGET_URL`: The URL of the endpoint to be load tested
+* `TEST_LOCATION_HEADER_NAME`: The of the HTTP header used to manually specify the location from which the request originated. This should match the value of CONTILE_LOCATION_TEST_HEADER on Contile
+
 The load tests were written using
 [Molotov](https://molotov.readthedocs.io/en/stable/) and can be started using
 the following command:
 
 ```sh
-$ molotov -c -v d <duration>
+$ molotov -c -v d <duration> -s <scenario name>
 ```
 
 where `<duration>` is the length of time in seconds that you want the load test
-to run.
+to run, and `<scenario name>` is one of the following:
+* `request_from_consistent_location_with_consistent_user_agent`: Requests all originate from the same location (USCA) and have the same user agent (Google Chrome on Windows)
+* `request_from_random_location_with_consistent_user_agent`: Requests originate from randomly-chosen locations but have the same user agent (Google Chrome on Windows)
+* `request_from_consistent_location_with_random_user_agent`: Requests originate from the same location but have randomly-chosen user agents
+* `request_from_random_location_with_random_user_agent`: Requests originate from a randomly-chosen location and have randomly-chosen user agents
```

Comment:
We may just need this one scenario, since it's closest to what we'd see in real life